src/validation.h

```diff
@@ -896,18 +866,53 @@ class ChainstateManager
     //! by the background validation chainstate.
     bool m_snapshot_validated{false};
 
+    CBlockIndex* m_best_invalid;
```

Comment:
I was just curious, though. How about naming this variable to:
```suggestion
    CBlockIndex* m_best_invalid_index;
```
or to:
```suggestion
    CBlockIndex* m_index_best_invalid;
```
Because:
- This would help express the purpose of this variable more clearly.
- Since this replaces `pindexBestInvalid`, it makes it more logical to name it this way.

Reply:
Alternatively `m_best_invalid_block_index`, though that's a bit long

Reply:
Personally I am not a fan of putting the type of the symbol in the name of the symbol.

For example, I think we prefer `std::string name` over `std::string strName` or `std::string name_string`.

Also, leaving as-is for now to not invalidate reviews.